README.md

```diff
@@ -25,8 +25,9 @@ Visualization of our network architecture: [[netron]](https://netron.app/?url=ht
     ```Shell
```

Comment:
I think 
```markdown
The source code is based on [FaceBoxes.PyTorch](https://github.com/sfzhang15/FaceBoxes.PyTorch) and [ssd.pytorch](https://github.com/amdegroot/ssd.pytorch).
```
can be deleted now.

Please update the link to the network architecture as well at line 7.